### PR TITLE
fix: UI of Input widget is not proper when icon is set and left aligned

### LINKED ISSUE #11198: [Bug]: UI of Input widget is not proper when icon is set and left aligned
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Screenshot 2022-02-16 at 12 23 50 PM

### Steps To Reproduce

1. Drag and drop an Input widget
2. In Icon property from property pane set an icon
3. Observe that the left border of the widget is not displayed.

### Environment

Production

### Version

Cloud

### PR DESCRIPTION
**Pull request by wmdev0808**
## Description

Steps To Reproduce
1. Drag and drop an Input widget
2. In Icon property from property pane set an icon
3. Observe that the left border of the widget is not displayed.

Fixes #11198 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Test A
- Test B

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>
No changes to code coverage between the base branch and the head branch</details>

### CODE REVIEW

File: app/client/src/widgets/BaseInputWidget/component/index.tsx [hunk 1]
```diff
@@ -205,7 +205,6 @@ const InputComponentWrapper = styled((props) => (
         top: 3px;
       }
       input:not(:first-child) {
-        border-left: 1px solid transparent;
```
Comment (somangshu) [inline app/client/src/widgets/BaseInputWidget/component/index.tsx:208]: @sbalaji1192 can you tell me what this was for?

The change has fixed the current issue, but not sure what else its impacting, you added this while working on the Input v2

**Review by somangshu — COMMENTED** [review]
Looking for some opinion

**Review by somangshu — APPROVED** [review]

**Comment by somangshu** [comment]
@wmdev0808 we need to get this confirmed before merging

File: app/client/src/widgets/BaseInputWidget/component/index.tsx (hunk 1 above)
Comment (sbalaji1192) [reply app/client/src/widgets/BaseInputWidget/component/index.tsx:208]: @somangshu IIRC, added this to fix a border issue in the phone input widget.  Just tested this on deployment, the phone input border looks fine. change looks good to me.

**Review by sbalaji1192 — APPROVED** [review]

**Pull request merged**